### ISSUE TITLE
Got an UI lag when mapped VAL and BASE when PRB=COUNT

### ISSUE THREAD

**Issue by vcvrackideas** [sebastien-bouffier/Bidoo#150]
OS (if applicable):win
Version (or "dev" if compiling from source):1.1.8
![2020-04-21_14-11-39](https://user-images.githubusercontent.com/42435368/79859861-5a574780-83da-11ea-8795-46cbea2ec4f4.png)
Knobs just disappear when I applied midi mapping to those parameters. Init doesn't help 
EDIT - same with DICE and value knob

**Comment by sebastien-bouffier** [comment]
Hi, I look into that. I have to check how knob visibility is managed (hidden vs. destroyed) and maybe I will have to change the way I test Prob. value because I don't know how snap buttons behave with midi mapping.

**Comment by sebastien-bouffier** [comment]
Issue found. Snap property (knob jumping between integers) is only handled at knob widget level and the midi mapping tool works at quantity level in the framework which does know its owner is snapped. I made changes in the way those params are handled in the module and thanks to you I discovered that module MIDI-MAP and I tested with my old mpd24 and it seems to work fine now. Available in the next release.
Cheers
